config/hooks/normal/0520-Qtpyvcp-repos.hook.chroot

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+curl sS https://repository.qtpyvcp.com/repo/kcjengr.key | gpg --dearmor | sudo tee /usr/share/keyrings/kcjengr.gpg
```

Comment:
What is 'sS' doing here?  It causes curl to report 'Could not resolve host: sS'.

I believe the key should be stored in /etc/apt/keyrings/, not under /usr/, as it is not part of a installed package.

Reply:
The curl format is as provided by the qtpyvcp guys here: https://kcjengr.github.io/probe_basic/quick_start_apt.html
I did not get any errors here on several tests and the repos installed for me.

The standard gpg method relies on the users' .gnupg folder for keys. I did a lot of research to find out how to bypass the users profile because no user is present on install.
I standardised on /usr because it was indicated that data /etc/apt/keyrings was blindly accepted by apt which could be a security risk. I am open to further guidance though.

Reply:
Ok, on further review, /etc/apt/keyrings/, is OK. I will revise locations.
Ref:  https://askubuntu.com/questions/1286545/what-commands-exactly-should-replace-the-deprecated-apt-key

Reply:
argh. Should have been -sS